prefix.go

```diff
@@ -699,7 +699,6 @@ func (p *Prefix) Usage() Usage {
 // with ten attempts and jitter delay ~100ms
 // returns only error of last failed attempt
 func retryOnOptimisticLock(retryableFunc retry.RetryableFunc) error {
-
```

Comment:
unrelated

Reply:
Yes, but then there should be a extra PR which introduces gofmt to the makefile

Reply:
Sure, I reverted those formatting changes.